carbon-design-system/ibm-security
# feat(progress-animation): add component with demo

## Affected issues

- Resolves #342 

## Proposed changes

- add `ProgressAnimation` component with light styles & demo

## Testing instructions

- check out the `ProgressAnimation` story in the deploy (https://deploy-preview-418--ibm-security.netlify.com/?path=/story/components-progressanimation--default) & play around with the knobs.
